frontend/src/streamlist/StreamsTable.jsx

```diff
@@ -105,14 +105,15 @@ function getIcons() {
 
 export default class StreamsTable extends React.Component {
   render() {
-    const { streamDetails } = this.props;
+    const { streams } = this.props;
+    console.log(streams);
```

Comment:
is this `console.log` necessary?

Reply:
Oh woops, no, just some left over from local testing.